packages/astro/src/runtime/server/hydration.ts

```diff
@@ -166,5 +166,11 @@ export async function generateHydrateScript(
 		})
 	);
 
+	['data-astro-transition-scope', 'data-astro-transition-persist'].forEach(name => {
```

Comment:
nit: we could move this array in a top-level `const` variable, so we don't create it every single time